src/edu/washington/escience/myria/operator/CSVFileScanFragment.java

```diff
@@ -179,30 +179,36 @@ public CSVFileScanFragment(
   @Override
   protected TupleBatch fetchNextReady() throws IOException, DbException {
     long lineNumberBegin = lineNumber;
+    boolean unfinishedEncapsulatedTokenRow = false;
```

Comment:
Personally, I would have no idea what "encapsulated token" meant if I hadn't looked at the `CSVParser` source. How about a clearer name like `truncatedQuotedField` or the like?